### PR TITLE
[Naitei] ft:View_Product_Admin

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine
https://edu-redmine.sun-asterisk.vn/issues/78534
## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

image

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/assets/stylesheets/admin_base.scss [hunk 1]
```diff
@@ -2,10 +2,15 @@
...
 
+.wrapper{
+  width: 100%;
+}
+
 .admin__body{
   display: flex;
 }
 
+
```
Comment (TranMinhHuy2003) [inline app/assets/stylesheets/admin_base.scss]: xóa dòng dư

File: app/views/admin/products/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      <tr>
+        <th><%= t("admin.view.id") %></th>
+        <th><%= t("admin.view.name") %></th>
+        <th><%= t("admin.view.description") %></th>
+        <th><%= t("admin.view.price") %></th>
+        <th><%= t("admin.view.remain_quantity") %></th>
+        <th><%= t("admin.view.created_at") %></th>
+        <th><%= t("admin.view.updated_at") %></th>
+        <th><%= t("admin.view.actions") %></th>
+      </tr>
```
Comment (TranMinhHuy2003) [inline app/views/admin/products/index.html.erb]: mấy cái i18n bỏ bớt () ko cần thiết

**Review by TranMinhHuy2003 — COMMENTED** [review]
cmt

File: app/views/admin/products/index.html.erb (hunk 2 above)
Comment (SuKay712) [reply app/views/admin/products/index.html.erb]: Đừng view ID ra table, xài số thứ tự cho table

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/views/admin/products/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <th><%= t "admin.view.description" %></th>
+        <th><%= t "admin.view.price" %></th>
+        <th><%= t "admin.view.remain_quantity" %></th>
+        <th><%= t "admin.view.created_at" %></th>
+        <th><%= t "admin.view.updated_at" %></th>
+        <th><%= t "admin.view.actions" %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= render partial: "product", collection: @products, as: :product%>
```
Comment (ghost) [inline app/views/admin/products/index.html.erb]:
```suggestion
      <%= render partial: "product", collection: @products, as: :product %>
```

**Review by ghost — COMMENTED** [review]
fix nốt rồi anh merged

**Comment by ThaiKhacDuoc** [comment]
ready

**Pull request merged**